### PR TITLE
Dafna/add table header to requests

### PR DESCRIPTION
**Pull request by dafna-bashan**

### CODE REVIEW

File: src/ui/assignments/src/components/filters/ProductFilter.tsx [hunk 1]
```diff
@@ -61,16 +63,32 @@ export const ProductFilter: React.FunctionComponent<ProductFilterProps> = ({
     _setSelectedProducts(newSelectedProducts);
   };
 
+  const comboboxRef = useRef<HTMLInputElement>(null)
+
+  const handleResize = () => {
```
Comment (ChengaDev) [inline src/ui/assignments/src/components/filters/ProductFilter.tsx]: Hey @dafna-bashan , it seems like the code from [this PR](https://github.com/giltayar/restart-orders/pull/4), maybe you did not checked out from main?

Comment (dafna-bashan) [reply src/ui/assignments/src/components/filters/ProductFilter.tsx]: Maybe, do you know how can i solve it? @ChengaDev

Comment (ChengaDev) [reply src/ui/assignments/src/components/filters/ProductFilter.tsx]: After you merge the second PR, do a merge from the main to this branch and push, that would leave only the relevant code in this PR.

Comment (dafna-bashan) [reply src/ui/assignments/src/components/filters/ProductFilter.tsx]: I'm not sure I understood you correctly. Do i need to merge this branch to the main branch before you review the code?

**Review by ChengaDev — APPROVED** [review]

**Pull request merged**